src/reader/parser.rs

```diff
@@ -19,21 +19,21 @@ use memchr;
 /// get back produced [`Event`]s.
 #[derive(Clone)]
 pub(super) struct Parser {
-    /// current buffer position, useful for debugging errors
+    /// Number of bytes read from the source of data since the parser was created
```

Comment:
Which is not going to work once we support multiple encodings, unfortunately.

Reply:
We can change it when we changes the working with encodings